marconi/src/Marconi/Index/Utxo.hs

```diff
@@ -163,9 +170,10 @@ onInsert _ix _update = pure []
 
 toRows :: UtxoUpdate -> [UtxoRow]
 toRows update = update ^. outputs
-  & map (\(out, ref)  -> UtxoRow { _address   = Ledger.txOutAddress out
-                                 , _reference = ref
-                                 })
+  & map (\(C.TxOut addr _ _ _, ref) ->
+        UtxoRow { _address   = addr
+                , _reference = ref
+                })
 
 onlyAt :: Address -> UtxoRow -> Bool
-onlyAt address' row = address' == (row ^. address)
+onlyAt address' = has $ address . only address'
```

Comment:
I think here the original version was clearer. I don't know what `has` and `only` do so to know it I'd have to go and look at lens documentation and even then it will take a bit. The original version on the other hand pretty much says what it does.

Reply:
No problem, I have this bad habit of putting everything in one lens when I see one usage of them in an expression, I'm reverting it.